.env

```diff
@@ -17,4 +17,15 @@ REACT_APP_API_BACKEND_AUDIENCE="exampleapp-backend"
 REACT_APP_OIDC_TOKEN_EXCHANGE_PATH="/api-tokens/"
 REACT_APP_PROFILE_BACKEND_URL="https://profiili-api.test.kuva.hel.ninja/graphql/"
 REACT_APP_PROFILE_AUDIENCE="https://api.hel.fi/auth/helsinkiprofile"
-REACT_APP_PROFILE_UI_URL="https://profiili.test.kuva.hel.ninja"
\ No newline at end of file
+REACT_APP_PROFILE_UI_URL="https://profiili.test.kuva.hel.ninja"
+REACT_APP_PLAIN_KEYCLOAK_URL="https://tunnistus.dev.hel.ninja/auth"
+REACT_APP_PLAIN_KEYCLOAK_CLIENT_ID="exampleapp-ui"
+REACT_APP_PLAIN_KEYCLOAK_SCOPE="openid profile email https://api.hel.fi/auth/helsinkiprofiledev"
+REACT_APP_PLAIN_KEYCLOAK_REALM="helsinki-tunnistus"
+REACT_APP_PLAIN_KEYCLOAK_LOGOUT_PATH="/"
+REACT_APP_PLAIN_KEYCLOAK_RESPONSE_TYPE="id_token token"
+REACT_APP_PLAIN_KEYCLOAK_SILENT_AUTH_PATH="/silent_renew.html"
+REACT_APP_PLAIN_KEYCLOAK_CALLBACK_PATH="/callback_kc/"
+REACT_APP_PLAIN_KEYCLOAK_AUTO_SIGN_IN="false"
+REACT_APP_PLAIN_KEYCLOAK_LOGGING="false"
+REACT_APP_PLAIN_KEYCLOAK_AUTO_SILENT_RENEW="true"
```

Comment:
Would be good to not have the "KEYCLOAK" word in these names either. Perhaps it's "SUOMIFI" then.

Reply:
Renamed